build.ps1

```diff
@@ -6,5 +6,9 @@ function exec([string] $cmd) {
     }
 }
 
+# After first v3 release, $version is just `git describe`
+$version = & { git describe --always }
+$version = "3.0.0-preview1-$version"
```

Comment:
why `preview1`?

Reply:
The naming conversion used by dotnet and aspnet. Need to ship several previews before shipping v3.